hw/drivers/sensors/bma253/src/bma253.c

```diff
@@ -2714,65 +2987,114 @@ bma253_set_fifo_cfg(const struct bma253 * bma253,
         break;
     }
 
+    data |= (0x03 << 2);
+
     return set_register((struct bma253 *)bma253, REG_ADDR_FIFO_CONFIG_1, data);
 }
 
-int
-bma253_get_fifo(const struct bma253 * bma253,
-                enum bma253_g_range g_range,
-                enum fifo_data fifo_data,
-                struct accel_data * accel_data)
+
+struct sensor_data_subscriber_info {
```

Comment:
This structure is not needed. There is a reason we have:
```
 644 struct sensor_read_ctx {
 645     sensor_data_func_t user_func;
 646     void *user_arg;
 647 };  
```
We want to be able to abstract app from the driver. Please let it be that way.

Reply:
Can you please check how we handle FIFOs in `hw/drivers/sensors/lis2dh12/src/lis2dh12.c`

Reply:
agree, we did not notice there was a similar structure needed, thanks for the info, and we will reuse sensor_read_ctx.